x-pack/plugins/uptime/server/lib/requests/search/refine_potential_matches.ts

```diff
@@ -145,9 +143,6 @@ export const query = async (
                     top: {
                       top_hits: {
                         sort: [{ '@timestamp': 'desc' }],
-                        _source: {
-                          includes: ['monitor.check_group', '@timestamp'],
```

Comment:
any reason why this is removed? keeping 

`_source: ['@timestamp'],`

seems like a good thing here.

Reply:
Good call! Adding back in.